qt/scientific_interfaces/Inelastic/QENSFitting/FitDataView.cpp

```diff
@@ -32,10 +32,11 @@ namespace MantidQt {
 namespace CustomInterfaces {
 namespace Inelastic {
 
-FitDataView::FitDataView(QWidget *parent) : FitDataView(defaultHeaders(), parent) {}
+FitDataView::FitDataView(QWidget *parent, std::string const &tabName)
+    : FitDataView(defaultHeaders(), parent, tabName) {}
 
-FitDataView::FitDataView(const QStringList &headers, QWidget *parent)
-    : QTabWidget(parent), m_uiForm(new Ui::FitDataView) {
+FitDataView::FitDataView(const QStringList &headers, QWidget *parent, std::string const &tabName)
+    : QTabWidget(parent), m_uiForm(new Ui::FitDataView), m_tabName(tabName) {
```

Comment:
Perhaps we can use some `QWidget` properties (like `windowTitle`) to store the tab name instead of a private member?